src/test/cypress/cypress/integration/Monitoring.spec.js

```diff
@@ -107,4 +108,289 @@ describe ('Monitoring screen tests',function () {
         cy.loadTestConf();
         cy.reload();
     })
+
+
+
+    it('Check monitoring cards reception', function () {
+
+        cy.loginOpFab('operator1','test');
+        cy.send6TestCards();
```

Comment:
Send 6 cards only once instead of in each it() (in the before method) , it will save time